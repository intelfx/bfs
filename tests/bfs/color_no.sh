LS_COLORS="no=01;92:" bfs_diff rainbow -color
