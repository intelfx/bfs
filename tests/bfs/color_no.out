[01;34m$'rainbow/\e[1m'[0m
[01;34m$'rainbow/\e[1m/'[0m[01;92m$'\e[0m'[0m
[01;34mrainbow[0m
[01;34mrainbow/[0m[01;32mexec.sh[0m
[01;34mrainbow/[0m[01;35msocket[0m
[01;34mrainbow/[0m[01;36mbroken[0m
[01;34mrainbow/[0m[01;36mchardev_link[0m
[01;34mrainbow/[0m[01;36mlink.txt[0m
[01;34mrainbow/[0m[01;92mfile.dat[0m
[01;34mrainbow/[0m[01;92mfile.txt[0m
[01;34mrainbow/[0m[01;92mlower.gz[0m
[01;34mrainbow/[0m[01;92mlower.tar[0m
[01;34mrainbow/[0m[01;92mlower.tar.gz[0m
[01;34mrainbow/[0m[01;92mlu.tar.GZ[0m
[01;34mrainbow/[0m[01;92mmh1[0m
[01;34mrainbow/[0m[01;92mmh2[0m
[01;34mrainbow/[0m[01;92mul.TAR.gz[0m
[01;34mrainbow/[0m[01;92mupper.GZ[0m
[01;34mrainbow/[0m[01;92mupper.TAR[0m
[01;34mrainbow/[0m[01;92mupper.TAR.GZ[0m
[01;34mrainbow/[0m[30;42msticky_ow[0m
[01;34mrainbow/[0m[30;43msgid[0m
[01;34mrainbow/[0m[33mpipe[0m
[01;34mrainbow/[0m[34;42mow[0m
[01;34mrainbow/[0m[37;41msugid[0m
[01;34mrainbow/[0m[37;41msuid[0m
[01;34mrainbow/[0m[37;44msticky[0m
